<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="layout_niveles">
      <item>
       <widget class="QPushButton" name="l1">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="l2">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="l3">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="l4">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="l5">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="l6">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="l7">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="l8">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="l9">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="l10">
        <property name="styleSheet">
         <string notr="true">height:40px;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,200);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/recursos/boys.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 120 oblique 48pt &quot;Keter YG&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#729fcf;&quot;&gt;Matem√°ticas Divertidas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/recursos/informacion.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="salir">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/recursos/boton_salir.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="respuesta1">
        <property name="styleSheet">
         <string notr="true">height:80%;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,250);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="respuesta2">
        <property name="styleSheet">
         <string notr="true">height:80%;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,250);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="respuesta3">
        <property name="styleSheet">
         <string notr="true">height:80%;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,250);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="respuesta4">
        <property name="styleSheet">
         <string notr="true">height:80%;
width:40px;
border-radius:15px;
background-color: rgb(173, 127, 168,250);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="sumas">
        <property name="styleSheet">
         <string notr="true">height:60px;
width:60px;
border-radius:25px;
background-color: rgb(173, 127, 168);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>sumas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restas">
        <property name="styleSheet">
         <string notr="true">height:60px;
width:60px;
border-radius:25px;
background-color: rgb(173, 127, 168);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>restas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="multiplicacion">
        <property name="styleSheet">
         <string notr="true">height:60px;
width:60px;
border-radius:25px;
background-color: rgb(173, 127, 168);
color:white;
font-size:20px;
font:bold;</string>
        </property>
        <property name="text">
         <string>multiplicacion</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="minijuego">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="meta">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#3465a4;&quot;&gt;META HOY : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="tiempo">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#3465a4;&quot;&gt;TIEMPO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%p%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="operacion">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>2000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">height:300px;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#4e9a06;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
